Makefile

```diff
@@ -455,7 +455,7 @@ generate-go-conversions-docker-infrastructure: $(CONVERSION_GEN) ## Generate con
 generate-go-openapi: $(OPENAPI_GEN) $(CONTROLLER_GEN) ## Generate openapi go code for runtime SDK
 	@mkdir -p ./tmp/sigs.k8s.io; ln -s $(ROOT_DIR) ./tmp/sigs.k8s.io/; cd ./tmp; \
 	for pkg in "api/v1beta1" "$(EXP_DIR)/runtime/hooks/api/v1alpha1"; do \
-		$(MAKE) clean-generated-openapi-definitions SRC_DIRS="./$${pkg}"; \
+		$(MAKE) clean-generated-openapi-definitions -C sigs.k8s.io/cluster-api SRC_DIRS="./$${pkg}"; \
```

Comment:
I noticed the `generate-openapi-definitions` target complaining about no target existing for the clean operation.
```bash
make[2]: *** No rule to make target 'clean-generated-openapi-definitions'.  Stop.
```
This is because we `cd` into `tmp` in the previous step and there is no Makefile there. I have fixed this by passing the path to the symlinked cluster-api folder in the `make` command.

Reply:
Passing in `$(ROOT_DIR)` should have the same effect since they are both symlinked, but I chose the `tmp` one since that is the context of the future operations in this make target.